doc/luasnip.txt

```diff
@@ -3628,197 +3673,423 @@ GENERAL ~
+                wordTrig = snip.wordTrig and true or false,
+                regTrig = snip.regTrig and true or false,
+            }
+        end
+    <
+
+This function returns:
+
+- `{ [string]: T[] }` Table mapping filetypes to list of data returned by snip_info function.
+
+
+unlink_current()                                  *luasnip-api-unlink_current*
+
+Removes the current snippet from the jumplist (useful if LuaSnip fails to
+automatically detect e.g. deletion of a snippet) and sets the current node
+behind the snippet, or, if not possible, before it.
+
+
+jump(dir): boolean                                          *luasnip-api-jump*
+
+Jump forwards or backwards
+
+- `dir: (1|-1)` Jump forward for 1, backward for -1.
+
+This function returns:
+
+- `boolean` `true` if a jump was performed, `false` otherwise.
+
+
+jump_destination(dir): LuaSnip.Node             *luasnip-api-jump_destination*
+
+Find the node the next jump will end up at. This will not work always, because
+we will not update the node before jumping, so if the jump would e.g. insert a
+new node between this node and its pre-update jump target, this would not be
+registered. Thus, it currently only works for simple cases.
+
+- `dir: (1|-1)` `1`: find the next node, `-1`: find the previous node.
+
+This function returns:
+
+- `LuaSnip.Node` The destination node.
+
+
+jumpable(dir): boolean                                  *luasnip-api-jumpable*
+
+Return whether jumping forwards or backwards will actually jump, or if there is
+no node in that direction.
+
+- `dir: (1|-1)` `1` forward, `-1` backward.
+
+
+expandable(): boolean                                 *luasnip-api-expandable*
+
+Return whether there is an expandable snippet at the current cursor position.
+Does not consider autosnippets since those would already be expanded at this
+point.
+
+
+expand_or_jumpable(): boolean                 *luasnip-api-expand_or_jumpable*
+
+Return whether it’s possible to expand a snippet at the current
+cursor-position, or whether it’s possible to jump forward from the current
+node.
+
+
+in_snippet(): boolean                                 *luasnip-api-in_snippet*
+
+Determine whether the cursor is within a snippet.
+
+
+expand_or_locally_jumpable(): boolean *luasnip-api-expand_or_locally_jumpable*
+
+Return whether a snippet can be expanded at the current cursor position, or
+whether the cursor is inside a snippet and the current node can be jumped
+forward from.
+
+
+locally_jumpable(dir): boolean                  *luasnip-api-locally_jumpable*
+
+Return whether the cursor is inside a snippet and the current node can be
+jumped forward from.
+
+- `dir: (1|-1)` Test jumping forwards/backwards.
+
+
+snip_expand(snippet, opts?): LuaSnip.ExpandedSnippet *luasnip-api-snip_expand*
+
+Expand a snippet in the current buffer.
+
+- `snippet: LuaSnip.Snippet` The snippet.
+- `opts?: LuaSnip.Opts.SnipExpand?` Optional additional arguments. Valid keys
+    are:
```

Comment:
I think we should have the same forced-on-new-line for `Valid keys are`

Markdown looks like:
![image](https://github.com/user-attachments/assets/06c868b0-dca2-4af9-b708-b57e140a43f2)

Reply:
5be06e2a47f931dd5ec9978d1b6a9e5f98323f39